index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Build a Tribute Page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <main id="main">
+      <div id="title">
+        <h1>Dr. Norman Borlaug</h1>
+        <div>The man who saved a billion lives</div>
+      </div>
+      <figure id="img-div">
+          <img src="images/tribute_img.jpg" alt="" id="image">
+          <div id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger. </div>
```

Comment:
Please, use semantic HTML. Replace the `<div id="img-caption">` element with a `<figcaption>` element within the <figure> element to provide a caption for the image.

Reply:
- [ ] You did a great job to inform the student the element where semantic HTML should be used. However, to improve the review, let's consider stating the detailed reason to why the changes should be applied. This will help the student have a clear understanding of your request ☺️.

Reply:
I have updated my initial comment to this:
Please, use semantic HTML. Replace the `<div id="img-caption">` element with a `<figcaption>` element. The figcaption tag is a semantic tag that represents a caption or legend for a figure. Using it to wrap the image caption will make it more meaningful and help search engines and assistive technologies understand the context of the image.